receiver/splunkhecreceiver/receiver_test.go

```diff
@@ -18,6 +18,7 @@ import (
 	"time"
 
 	jsoniter "github.com/json-iterator/go"
+	"github.com/open-telemetry/opentelemetry-collector-contrib/internal/sharedcomponent"
```

Comment:
I think this import should move down, next to `"github.com/open-telemetry/opentelemetry-collector-contrib/internal/common/testutil"`

Reply:
thanks, fixing